site/pages/institution-landing-en.hbs

```diff
@@ -0,0 +1,188 @@
+---
+{
+"title": "Institution-landing-page",
+"language": "en",
+"altLangPrefix": "institution-landing",
+"pageType": "institution-landing",
+"pageclass": "page-type-nav",
+"breadcrumb": [
+{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+],
+"dateModified": "2021-03-31",
+"share": "true",
+"noMainContainer": true,
+"followUsLayout": "inline"
+}
+---
+
+<div class="provisional gc-institutional-bg mrgn-tp-md"
+	data-bgimg="https://design.canada.ca/coded-layout/images/ip-cover-image-1200x726.jpg">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+				<p>Short description of the institution’s mandate.</p>
+				<a class="btn btn-call-to-action" href="#">Super task button [optional]</a>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="bg-gctheme opct-90">
+	<div class="container">
+		<div class="row">
+			<div class="mrgn-tp-md mrgn-bttm-md col-xs-12">
+				<h2 class="wb-inv">Featured link</h2>
+				<p class="pull-left mrgn-bttm-0"><a class="stretched-link text-white" href="#"><b>Featured link [optional]</b></a></p>
+			</div>
+		</div>
+	</div>
+</div>
+
+<section class="well well-sm provisional gc-most-requested">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-2">
+				<h2>Most requested</h2>
+			</div>
+			<div class="col-md-10">
+				<ul class="colcount-md-2">
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+				</ul>
+			</div>
+		</div>
+	</div>
+</section>
```

Comment:
@delisma I included it, but was curious to know why is "Lato" chosen as the font family instead of the usual "Noto sans" for the ".gc-mostrequested" list? And do not forget that choosing "Lato" (if we consider this an option since I'm getting a little outside the requirements here @duboisp), then the Most requested component will have Lato all accross.

Reply:
@jmealing Should we file a different set of requirements for the `gc-mostrequested` as a stand-alone component?

Reply:
@delisma Right now there should only two Most Requested looks. One is on the homepage (no bullets, 4 columns of two), and the other Most requested look would be a component that could be added to other pages (with bullets)

"Lato" is already used for the Most requested on the home page, it was chosen at that time for the visual weight/compactness of the font family. Both Most requested lists, even with a different layout should use the same font

Reply:
@GormFrank let create a component